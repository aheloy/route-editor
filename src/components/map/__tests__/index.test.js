import React from 'react';

import Map from '../';


const dots = [
  {
    text: '1'
  },
  {
    text: '2'
  },
  {
    text: '3'
  }
];

const onChange = () => 0;

test('render a Map', () => {
  const wrapper = shallow(
    <Map
      dots={dots}
      onChange={onChange}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
