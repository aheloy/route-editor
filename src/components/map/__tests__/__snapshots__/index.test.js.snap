// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Map 1`] = `
<styled.div
  id="map"
/>
`;
